src/python/pcmdi/mean_climate_metrics_calculations_ExtInfo.py

```diff
@@ -0,0 +1,142 @@
+import cdms2 as cdms
```

Comment:
@gleckler1 @durack1 same question again. Do we want these calaculations as part of our repo?